monitoring/uss_qualifier/scd/executor/executor.py

```diff
@@ -73,17 +76,53 @@ def format_combination(combination: Dict[str, TestTarget]) -> List[str]:
 
 
 def run_scd_tests(locale: Locality, test_configuration: SCDQualifierTestConfiguration,
-                  auth_spec: str):
+                  auth_spec: str) -> bool:
     automated_tests = load_scd_test_definitions(locale)
     configured_targets = list(map(lambda t: TestTarget(t.name, t, auth_spec), test_configuration.injection_targets))
+    report = Report(
+            qualifier_version=os.environ.get("SCD_VERSION", "unknown"),
```

Comment:
It might be nice to have run_locally_scd populate this reasonably to get a more complete/accurate test of the system

Reply:
I am sorry, I don't understand the comment, could you please elaborate? Currently, this variable is set as an argument at [build time](https://github.com/swiss-foca/dss/blob/scd_automated_testing/reports/monitoring/uss_qualifier/run_locally_scd.sh#L52).

Reply:
In the report, I'm currently seeing `"qualifier_version": "interuss/uss_qualifier_scd/v0.0.0",` in the report.  When I wrote this comment, I was thinking I would see 0.1.0 somewhere, but upon further thought, I shouldn't see that because only `dss` is at 0.1.0.  But, I would still expect to see `-<commit hash>`.  That doesn't necessarily need to be solved in this PR though.

Reply:
👍 Solved by #719